### ISSUE TITLE
Accessibility: dark mode: link color too dark

### ISSUE THREAD

**Issue by bastienvoirin** [lilaq-project/lilaq-project.github.io#15]
Hello,

The link color appears too dark in dark mode: the contrast ratio between #5f36c7 and #1b1b1c is 2.32:1, whereas

> WCAG 2.0 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. WCAG 2.1 requires a contrast ratio of at least 3:1 for graphics and user interface components (such as form input borders). WCAG Level AAA requires a contrast ratio of at least 7:1 for normal text and 4.5:1 for large text.
> 
> Large text is defined as 14 point (typically 18.66px) and bold or larger, or 18 point (typically 24px) or larger.

The #9783c9 of `lq.` prefixes has a contrast ratio of 5.24:1, which passes AA but not AAA. This one doesn't need to change.

<img width="1066" height="1363" alt="Image" src="https://github.com/user-attachments/assets/050d7f0b-adb4-4073-b0f5-ed6fbeb8c57e" />

**Comment by Mc-Zen** [comment]
It should be fixed now! 

I also improved the color contrast of the link namespaces (e.g., `plot.` from `plot.stroke`) and defaults to reach AA compliance. 

The only exception now is the "(source)" link that is shown with each function signature. But this link is a a bit special, probably rarely used and nothing that is really _read_.
